recipes/cuda-version/meta.yaml

```
@@ -0,0 +1,33 @@
+{% set name = "cuda-version" %}
+{% set version = "12.0.0" %}
+{% set major_version = version.split('.')[0]|int %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  run_constrained:
+    - __cuda >={{ major_version }}
+
+test:
+  commands:
+    - exit 0
+
+about:
+  home: https://developer.nvidia.com/cuda-toolkit
+  #license_file: LICENSE
```

Since it would be downloaded in `source`, it should be possible to include it here

```suggestion
  license_file: LICENSE
```

---

Done below